### PR TITLE
Toggle zoom/pan off

### PR DESCRIPTION
**Pull request by bnmajor**
When the calibration line picker, polygon mask tool, or polar mask tool are selected make sure that the navigation bar `Zoom` and `Pan` options are toggled off.

Fixes: #552

### CODE REVIEW

File: hexrd/ui/image_tab_widget.py [hunk 1]
```diff
@@ -171,6 +171,14 @@ def update_ims_toolbar(self):
             self.toolbars[idx]['sb'].update_name(self.image_names[idx])
             self.toolbars[idx]['sb'].update_range(True)
 
+    def toggle_off_toolbar(self):
+        toolbars = [bars['tb'] for bars in self.toolbars]
+        for tb in toolbars:
+            if tb.mode == 'zoom rect':
+                tb.zoom()
+            if tb.mode == 'pan/zoom':
```
Comment (cjh1) [inline hexrd/ui/image_tab_widget.py:179]: Lets move these to the constants module, with suitable comment, as we are reaching into matplotlib internals!

Comment (bnmajor) [reply hexrd/ui/image_tab_widget.py]: Sounds good, updated. Let me know if the comment seems sufficient

**Review by cjh1 — APPROVED** [review]
LGTM

**Review by psavery — APPROVED** [review]
LGTM

**Pull request merged**